zqd/handlers_test.go

```diff
@@ -0,0 +1,121 @@
+package zqd_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zio/bzngio"
+	"github.com/brimsec/zq/zio/zngio"
+	"github.com/brimsec/zq/zng/resolver"
+	"github.com/brimsec/zq/zqd"
+	"github.com/brimsec/zq/zqd/api"
+	"github.com/brimsec/zq/zqd/space"
+	"github.com/brimsec/zq/zql"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Would you please add a test for the space listing api?

Reply:
Sure!